src/components/atoms/Button/ButtonGroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export interface ButtonGroupProps {
+  className?: string
+  style?: React.CSSProperties
```

Comment:
Is it a good thing to allow native style props when we already use styled-component?

Reply:
Yeah I guess it is. There are lots of edge cases where you just wanna have some inline-styles. So we decided to just allow this for each of our components.